.github/workflows/actions.yml

```diff
@@ -245,6 +245,7 @@ jobs:
             cmake \
             g++ \
             gettext \
+            gettext-dev \
```

Comment:
nit: I think all `<name>-dev` deps already depend on according `<name>` dep. So `gettext` can be removed.
I have already tried something like this, when build on alpine runner failed to find some headers.

Reply:
removed in 7e82bc9d20311bd3eac052c46aa222600a85cb0f